lib/client.c

```diff
@@ -1353,7 +1468,7 @@ SF_STATUS STDCALL _snowflake_execute_ex(SF_STMT *sfstmt,
     sf_bool success = SF_BOOLEAN_FALSE;
     uuid4_generate(sfstmt->request_id);
     URL_KEY_VALUE url_params[] = {
-        {.key="requestId=", .value=sfstmt->request_id, .formatted_key=NULL, .formatted_value=NULL, .key_size=0, .value_size=0}
+            {.key="requestId=", .value=sfstmt->request_id, .formatted_key=NULL, .formatted_value=NULL, .key_size=0, .value_size=0}
```

Comment:
Not a big deal, but wondering why you added another tab here?

Reply:
Just for consistency. There were two tabs for statements spanning multiple lines at other places   but I guess I haven't made that change everywhere. Will revert.

Reply:
You can keep this, I see it is the same below. I was just curious mostly.